crates/ruma-common/CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # [unreleased]
 
+Bug fixes:
+
+* Change default `invite` power level to `0`
+  * This spec was determined to be wrong about the default previously:
```

Comment:
```suggestion
  * The spec was determined to be wrong about the default:
```
I don't think `previously` adds anything since `was` already implies the past.